Changelog.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+### 1.2.0
+Image: 
+- k8s.gcr.io/ingress-nginx/controller:v1.2.0@sha256:d8196e3bc1e72547c5dec66d6556c0ff92a23f6d0919b206be170bc90d5f9185
+- k8s.gcr.io/ingress-nginx/controller-chroot:v1.2.0@sha256:fb17f1700b77d4fcc52ca6f83ffc2821861ae887dbb87149cf5cbc52bea425e5
+
+This major release fixes the recently announced CVE on busybox & ssl_client, in addition to making the chrooted ingress-nginx-controller (#8337), generally available to all users, as an option. This release also brings an important fix to a redirection related bug  #8468.
+
+Changes:
+- [8481](https://github.com/kubernetes/ingress-nginx/pull/8481) Fix log creation in chroot script
+- [8479](https://github.com/kubernetes/ingress-nginx/pull/8479) changed nginx base img tag to img built with alpine3.14.6
+- [8478](https://github.com/kubernetes/ingress-nginx/pull/8478) update base images and protobuf gomod
+- [8468](https://github.com/kubernetes/ingress-nginx/pull/8468) Fallback to ngx.var.scheme for redirectScheme with use-forward-headers when X-Forwarded-Proto is empty
+
```

Comment:
I think we should include what's in beta.0 and beta.1

Reply:
added other non-housekeeping Pas as well. Please check

Reply:
Please add the fact that this is a new minor due to breaking change, and saying about the chroot image and deep inspector as well (i did in beta.0 changelog)

Reply:
I edited the Changeling description as per your advise. Please check @rikatz

Reply:
The changelog is duplicated and seems missing some things, but I guess we can release and take care of it later